### PR TITLE
Take screenshots of failing Theia tests

### PR DESCRIPTION
**Pull request by t1m0thyj**
## Proposed changes

Take screenshots of failing Theia tests and save them in the "results/integration" folder which is archived by the GitHub workflow.

Hopefully this will help us to diagnose the problem causing several tests to occasionally have transient failures.

Example run with screenshots in the artifact: https://github.com/zowe/vscode-extension-for-zowe/actions/runs/3915784470

## Release Notes

N/A since only tests have been changed

### CODE REVIEW

**Comment by t1m0thyj** [comment]
Verified this works locally, but the workflow skips uploading the artifact when tests fail. Marking as draft until this is fixed.

File: packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts [hunk 1]
```diff
@@ -257,6 +280,8 @@ describe("Delete Profiles", () => {
         await driverChrome.clickOnZoweExplorer();
     });
 
+    afterEach(screenshotIfFailed(driverChrome));
```
Comment (traeok) [inline packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts:283]: Do we need `afterEach` multiple times in this `describe` block?

**Review by traeok — APPROVED** [review]
LGTM! Just left a question about one of the Jest functions

File: packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts (hunk 1 above)
Comment (t1m0thyj) [reply packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts:283]: I believe there is only one `afterEach` per `describe` block in this PR, but the way the GitHub diff collapses the lines may make it tricky to review. This `afterEach` applies to the "Delete Profiles" `describe` on line 274.

Comment (traeok) [reply packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts:283]: Ah, sorry I should have taken a closer look. GitHub was hiding the `describe` blocks in between each `afterEach` 🤦‍♂️ thanks for clarifying!

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**